src/Core/SettingsEnums.h

```diff
@@ -242,4 +242,13 @@ DECLARE_SETTING_ENUM(S3QueueAction)
 
 DECLARE_SETTING_ENUM(ExternalCommandStderrReaction)
 
+enum class DateTimeOverflowMode
```

Comment:
Minor: I know, other places in this file do it the same way, but let's use camel case instead of all uppercase (like we'd use for macros):

```cpp
    Throw,
    Ignore,
    Saturate
```